zuul.d/project-templates.yaml

```diff
@@ -94,6 +94,22 @@
       The default set of functional test jobs for the OpenStack Charms
     check:
       jobs:
+        - noble-caracal:
+            voting: false
+            branches:
+              - main
+              - master
+              - stable/2024.1  # OpenStack
```

Comment:
I may be misunderstanding this one, but should it be `stable/2024.2` for bobcat rather than antelope?

Reply:
I'm confused with your comment. This is the noble-caracal job, so it should run for changes/patches proposed in the stable/2024.1 (caracal) branch

Reply:
yes, ignore my comment; I was misreading the dates.